web-test-runner.config.js

```diff
@@ -20,8 +38,47 @@ export default {
 					}
 				})
 			]
+		},
```

Comment:
I think we'll want to take all of this and turn it into a shared config that consumers can just import from `@brightspcae-ui/testing-helpers`.